robot_ws/src/hello_world_robot/launch/deploy_rotate.launch.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this
+# software and associated documentation files (the "Software"), to deal in the Software
+# without restriction, including without limitation the rights to use, copy, modify,
+# merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+# INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+# PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+# HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+# OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+"""Launch turtlebot3_bringup and a rotate node."""
+
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))  # noqa
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..', 'launch'))  # noqa
+
+import launch
+
+from launch_ros import get_default_launch_description
+import launch_ros.actions
+
+from ament_index_python.packages import get_package_share_directory
+
+import lifecycle_msgs.msg
+
+
+def generate_launch_description():
+    """Main."""
+    use_sim_time_false = launch.actions.DeclareLaunchArgument(
+        'use_sim_time',
+        default_value='false',
+        description='Use sim time',
+        )
+
+    turtlebot3_bringup_dir = get_package_share_directory('turtlebot3_bringup')
+    turtlebot3_bringup_launch = launch.actions.IncludeLaunchDescription(
+        launch.launch_description_sources.PythonLaunchDescriptionSource(
+            turtlebot3_bringup_dir + '/launch/robot.launch.py'))
+    
+    rotate_node = launch_ros.actions.Node(
```

Comment:
Feels like we should find a way to import some common definitions from rotate.launch.py instead of duplicating? Just a thought, not blocking.

Reply:
Agreed, I did not opt for this as it is not clear how argument passing through other launch files is standardized. Let's look into this a bit more as we move forward.